### ISSUE TITLE
[feature] CREPA coefficient decay

### ISSUE THREAD

**Issue by kabachuha** [bghira/SimpleTuner#2409]
CREPA is amazing for accelerating model convergence in my experience for image2video.

However, in the *text2video* mode, especially if you train against a solid background, it creates horizontal and vertical *stripes* and *washed out* feel.

I think a step-scheduled coefficient (maybe even with cutoff) for the regularization multiplier decay would improve this and make the end text2video results sharper.

**Comment by bghira** [comment]
interestingly we could have layer we create and optimise on timestep-specific values to use for this, and discard it after training. but yeah, a simple schedule would be the simplest.

**Comment by kabachuha** [comment]
Another way to determine the cutoff would be to set some threshold for the similarity value. As you know if you tested CREPA, the loss / similarity curve reach saturation after some time (and this time is often unknown at the start of the training run, it can take an hour or five hours). This similarity-based coefficient cutoff could be the most efficient way because it won't require the user to "guess it" when this curve plateaus and the model overfits on the deep encoder features
